integration_tests/integration/court-register.spec.ts

```diff
@@ -18,15 +19,15 @@ context('Court register', () => {
         courtId: 'SHFCC',
         courtName: 'Sheffield Crown Court',
         courtDescription: 'Sheffield Crown Court - Yorkshire',
-        type: { courtType: 'CROWN', courtName: 'Crown' },
```

Comment:
These codes in the mock data have to match the mock Court Types data, hence the change